### PR TITLE
Migrate authN tests

### PR DESCRIPTION
**Pull request by atharvar28**
Link to JIRA ticket if there is one: [GQE-278](https://ctds-planx.atlassian.net/browse/GQE-278)

### Deployment changes

[GQE-278]: https://ctds-planx.atlassian.net/browse/GQE-278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jacob50231 — APPROVED** [review]
This all looks good! Maybe take a look at the changes you made to logging in the get_tokens function in the ras.py file to make sure it's what was intended (I see you removed logging for the input json data on the post request)  but other than that this should be good to merge!

**Review by AlbertSnows — DISMISSED** [review]
no major comments. I am not a maintainer of vigil and thus my review may still miss things. if you can get more eyes on this before merge all the better.

**Comment by krishnaa05** [comment]
> This all looks good! Maybe take a look at the changes you made to logging in the get_tokens function in the ras.py file to make sure it's what was intended (I see you removed logging for the input json data on the post request) but other than that this should be good to merge!

Logger statements are now removed.

**Pull request merged**
